distributions/openhab/src/main/resources/userdata/etc/org.apache.karaf.log.cfg

```
@@ -20,4 +20,4 @@ size = 500
 # The pattern used to format the log statement when using log:display. This pattern is according
 # to the log4j layout. You can override this parameter at runtime using log:display with -p.
 #
-pattern = %d{HH:mm:ss.SSS} [%-5.5p] [%-37.37c] - %m%n
+pattern = %d{HH:mm:ss.SSS} [%-5.5p] [%-37.37c] - %h{%m%n}\n
```

Perhaps this will still work and prevent issues on Windows?

```suggestion
pattern = %d{HH:mm:ss.SSS} [%-5.5p] [%-37.37c] - %h{%m}%n
```

---

Thanks, seem to work nicely.